ci_framework/roles/openshift_provisioner_node/tasks/add_bridges.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create the external network connection
+  become: true
+  community.general.nmcli:
+    type: bridge
+    conn_name: "{{ cifmw_opn_external_bridge_name }}"
+    stp: false
+    state: present
+
+- name: Verify external network interface is defined
+  when:
+    - cifmw_opn_external_network_iface is undefined
+    - not ( cifmw_opn_dry_run | default(false)) | bool
```

Comment:
it would make more sense to put that `cifmw_opn_dry_run` parameter in the defaults/main.yml and just rely on its value.

Reply:
Thanks for the pointer. Yes, will make the necessary changes in the next patch set.